docs/devGuide/writingComponents.md

```diff
@@ -0,0 +1,115 @@
+{% set title = "Writing Components" %}
+<span id="title" class="d-none">{{ title }}</span>
+
+<frontmatter>
+  title: "{{ title }}"
+  layout: devGuide.md
+  pageNav: default
+</frontmatter>
+
+# {{ title }}
+
+<div class="lead">
+
+This page explains various concerns related to MarkBind components, focused on implementation and testing.
+</div>
+
```

Comment:
I think we can include a quick introduction here of 
1. what 'components' refer to & what a 'node' is.
1. since you use the words slot and attribute later on, maybe we can put in a quick description of the meaning now? In case people read it without reading through the whole component guide or need a refresher
1. The node processing workflow (explained in paragraphs 2 and 3  under transforming the node directly), since it seems applicable to all components and not just that part?

Reply:
Thanks for the suggestion! I think explaining node processing is a good idea since it's pretty important, will add it to the guide :o

Reply:
Yay, really helpful introduction!